content/en/docs/ops/ambient/usage/waypoint/index.md

```diff
@@ -0,0 +1,400 @@
+---
+title: L7 - waypoint proxy Layer
+description: User guide for L7 Processing.
+weight: 2
+owner: istio/wg-networking-maintainers
+test: n/a
+---
+
+{{<tip>}}
+Before you start with this guide, please make sure that you have already read the Ztunnel Networking sub-guide and have a basic understanding of Istio Ambient Mesh. This guide assumes that you have already completed the Installation, set up the ztunnel and enabled mTLS.
```

Comment:
possible to have link in tip?

would like to have link to Ztunnel Networking sub-guide or Installation, set up the ztunnel and enabled mTLS